.vscode/settings.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
Why this change?

Reply:
by mistake. I hope it fixed.

Reply:
the file is still being deleted in the PR... can you just restore it?

Reply:
done.